external-import/mandiant/Dockerfile

```diff
@@ -1,16 +1,12 @@
 FROM python:3.10-alpine
 
-# Copy the connector
-COPY src /opt/opencti-connector-mandiant
+COPY src/requirements.txt /tmp/
```

Comment:
You could use only one `COPY` instruction, the other `COPY` on row 9 will copy the requirements as well.

Reply:
I know, but the build time will drastically increase. I guess you see why ;)